libs/api/protocol/src/lib/core/command/base-create-communication-message.command.ts

```diff
@@ -0,0 +1,14 @@
+import { AuthUser } from '@kordis/shared/model';
+
+import {
+	RegisteredUnit,
+	UnknownUnit,
+} from '../entity/partials/unit-partial.entity';
+
+export interface BaseCreateCommunicationMessageCommand {
```

Comment:
```suggestion
export interface BaseCreateMessageCommand {
```
On naming:
Every entry is a protocol entry. However, there is a subset of protocol entries which document messages between two units. Those are called `message` entries. They are characterized by having a recipient and a channel in addition to the fields of a protocol entry. As such, I suspect this interface being for them, right?
`CommunicationMessage` is one instance of these message entries. Besides that there are other message entry types like signing on a rescue station or starting an operation. Basically every bi-directional communication.
On the other hand, we have protocol entires which are outside of this message subset. The typical one here being the status change which is a broadcast.